.travis.yml

```diff
@@ -12,7 +12,7 @@ before_install:
   - source script/install_phantomjs.sh
   - gem update --system # use the very latest Rubygems
   - rvm @global do gem uninstall bundler -a -x
-  - rvm @global do gem install bundler -v 1.14.2 # latest version known to work
+  - rvm @global do gem install bundler -v 1.14.3 # latest version known to work
```

Comment:
Well, I introduced this because of the same reason I proposed to commit lock files. https://github.com/activeadmin/activeadmin/pull/4744 was broken because of a buggy bundler release. @varyonic [pointed out](https://github.com/activeadmin/activeadmin/pull/4744#issuecomment-274338610) that it was too bad that our build breaks because of that, so I agreed and locked the version in https://github.com/activeadmin/activeadmin/pull/4744/commits/482734534adb69ca4eb08c889bfb810708f3ac8f.